optuna/visualization/matplotlib/_slice.py

```diff
@@ -15,6 +15,7 @@
 from optuna.visualization.matplotlib._matplotlib_imports import _imports
 from optuna.visualization.matplotlib._utils import _is_categorical
```

Comment:
Could you remove this import since `_is_categorical` is no longer used in this file?